recap/converters/postgresql.py

```diff
@@ -24,7 +25,15 @@
 
 
 class PostgresqlConverter(DbapiConverter):
-    def __init__(self, namespace: str = DEFAULT_NAMESPACE) -> None:
+    def __init__(
+        self,
+        ignore_array_dimensionality: bool = True,
```

Comment:
I wonder if the negation `support_array_dimensionality = False` or `include_array_dimensions = False` would be more intuitive for the user

Reply:
My vote is for 'enforce_array_dimensions'